inc/ScaffoldPackageCommand.php

```diff
@@ -110,6 +110,8 @@ public function package( $args, $assoc_args ) {
 		if ( ! Utils\get_flag_value( $assoc_args, 'skip-readme' ) ) {
 			WP_CLI::run_command( array( 'scaffold', 'package-readme', $package_dir ), array( 'force' => $force ) );
 		}
+
+		WP_CLI::run_command( array( 'package', 'install', $package_dir ) );
```

Comment:
Could we potentially use `WP_CLI::launch_self()` here, so installation happens as a part of a new process?